lumicks/pylake/tests/test_channels.py

```diff
@@ -534,7 +534,7 @@ def test_downsampling_over_subset():
     s = channel.Slice(channel.Continuous(d, 0, 10))
 
     sd = s.downsampled_over([(20, 40), (40, 60), (60, 80)])
-    np.testing.assert_allclose(sd.data, [(3+4)/2, (4+5)/2, (5+6)/2])
+    np.testing.assert_allclose(sd.data, [(3+4)/2, (5+6)/2, (7+8)/2])
     np.testing.assert_allclose(sd.timestamps, [(20+30)/2, (40+50)/2, (60+70)/2])
```

Comment:
OK, this one I don't get. There seems to be an obvious relation between `sd.data` and `sd.timestamps`, which was previously only correct for the first tuple of `(20, 40)`? Perhaps good idea to explain it in a brief comment?

Reply:
I've added a small comment.